tests/governance.py

```diff
@@ -212,6 +212,37 @@ def test_member_data(network, args):
     return network
 
 
+@reqs.description("Check /gov/members endpoint")
+def test_all_members(network, args):
+    primary, _ = network.find_primary()
+
+    with primary.client() as c:
```

Comment:
Can we check that this is accessible in recover/partOfPublicNetwork mode?

Reply:
I've manually confirmed it's accessible. I don't see an obvious way to fit that into our existing tests. Could add a new test case which does a recovery, or pass a list of functions (additional test cases) to run on the mid-recovery network to `test_recover_service`... I'll take a look.

Reply:
Now done.